src/reports/controllers/reportsController.js

```diff
@@ -87,3 +89,40 @@ module.exports.postStats = async (req, res, next) => {
     }
     return res.status(204).json();
 };
+
+module.exports.getExportedReport = async(req, res, next) => {
+    let exportedReport;
+    try{
+        let reportInput;
+        try {
+            reportInput = await aggregateReportGenerator.createAggregateReport(req.params.test_id, req.params.report_id);
+        } catch (err) {
+            return next(err);
+        }
+        exportedReport = await reports.exportReport(reportInput,req.params.file_format);
+    } catch (err){
+        return next(err);
+    }
+    return res.send(exportedReport);
+}
+
+module.exports.getExportedCompareReport = async(req,res,next) => {
+    let exportedCompareReport;
+    try{
+        let aggregateReportArray =[];
+        let reportIds = req.query['reportIds'].split(",");
+        let testIds = req.query['testIds'].split(",");
+        for (let index in reportIds){
+            try {
+                let result = await aggregateReportGenerator.createAggregateReport(testIds[index],reportIds[index]);
```

Comment:
wondering what will happen here if of some report isn't found.. will it return a good error message?
I guess tests will tell us :) (this comment is valid also for the single report csv